src/components/Header/context/patronContext.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+import { PatronContext, PatronProvider } from "./patronContext";
+
+describe("PatronContext", () => {
+  const PatronTestComponent = () => {
+    const name = React.useContext(PatronContext);
+    return <p>{name}</p>;
+  };
+
+  it("passes down the patron name value down to its children", () => {
```

Comment:
Maybe overkill, but should we test when the patronName is not provided, or is an empty string?

Reply:
Nope, that's a solid update.